### PR TITLE
moves v1.27.1 into main and bumps back to v1.28.0

### PR DESCRIPTION
**Pull request by johnclary**
## Associated issues
Companion to the production patch:
- https://github.com/cityofaustin/atd-moped/pull/1075

This brings main back in line with production and bumps us back to v1.28. 

## Testing
**URL to test:** 

**Steps to test:**

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable

### CODE REVIEW

File: moped-editor/src/auth/user.js [hunk 1]
```diff
@@ -109,7 +109,6 @@ export const initializeUserDBObject = (userObject) => {
     }).then((res) => {
       // Then we parse the response
       res.json().then((resData) => {
-        // If we have any user data,  use it
```
Comment (johnclary) [inline moped-editor/src/auth/user.js:112]: i guess [this patch commit](https://github.com/cityofaustin/atd-moped/pull/1076/commits/9a312d00c38d242a6b5c25e4102e26ac63f8aa1b) never made it's way back to main

**Review by frankhereford — APPROVED** [review]
Looks good to me - thank you for cleaning this up and keeping us in sync!

**Review by mddilley — APPROVED** [review]
🔄 🛳️

**Comment by johnclary** [comment]
thanks y'all!

**Pull request merged**